src/backend/feed/processor.js

```diff
@@ -107,6 +107,7 @@ function articlesToPosts(articles, feed) {
     articles.map(async (article) => {
       try {
         await Post.createFromArticle(article, feed);
+        //Add a debug log to see what is thrown?
```

Comment:
We end up logging this in the caller on line 214, and that one could probably be turned into a warning vs. error.